test/integration/scheduler/scheduler_test.go

```diff
@@ -538,4 +548,50 @@ var _ = ginkgo.Describe("Scheduler", func() {
 
 		framework.ExpectWorkloadsToBeAdmitted(ctx, k8sClient, prodClusterQ.Name, smallWl1, smallWl2)
 	})
+
+	ginkgo.It("Should schedule jobs borrowing quota from ClusterQueues in the same Cohort", func() {
+		job1 := testing.MakeJob("job1", ns.Name).Queue(prodBEQueue.Name).Request(
+			corev1.ResourceCPU, "11").Obj()
+		gomega.Expect(k8sClient.Create(ctx, job1)).Should(gomega.Succeed())
+		createdJob1 := &batchv1.Job{}
+		gomega.Consistently(func() *bool {
```

Comment:
use https://github.com/kubernetes-sigs/kueue/blob/425187b36a282851e08ad25d0fd51fc0ee5fc2d9/test/integration/framework/framework.go#L151-L168,
which should be faster

Reply:
only if the test is changed to use QueuedWorkload, otherwise we still have to refactor it again.

Reply:
Yes. This function may be added after I implemented this pr.
I refactor this again to use `ExpectWorkloadsToBePending`